README.md

```diff
@@ -21,15 +25,21 @@ First run (generates empty config that you need to fill out):
 ```sh
 poetry run aw-watcher-spotify
 ```
+### Step 1: Using pip
+Install the requirements:
 
-If this is the first time you run it on your machine, it will give you an error, this is normal. Just fill in the config file referenced in the error. You can create the client credentials needed at the [Spotify Developer portal](https://beta.developer.spotify.com/). In the dashboard, go to your app settings and add `http://localhost/` in the Redirect URIs section.
-
-Now run it again:
+```sh
+pip install -r requirements.txt
+```
```

Comment:
There's no need to have both a `requirements.txt` and a `pyproject.toml` (the former will inevitably get out-of-date).

To install the package with pip alone, using the deps specced in `pyproject.toml`, you can simply do `pip install .`

Reply:
Oh right! Thanks for that. I have pushed some more changes :)